Error due to operator "/" for strings

I am seeing this error:

```
libsvmdata/tests/test_core.py:3: in <module>
    from libsvmdata import fetch_dataset, print_supported_datasets
/nix/store/ps64ljvkg2qh00vk6iina3hb293xlvlk-python3-3.12.9-env/lib/python3.12/site-packages/libsvmdata/__init__.py:1: in <module>
    from libsvmdata.datasets import fetch_libsvm, download_libsvm
/nix/store/ps64ljvkg2qh00vk6iina3hb293xlvlk-python3-3.12.9-env/lib/python3.12/site-packages/libsvmdata/datasets.py:9: in <module>
    from libsvmdata.libsvm import DATASETS as libsvm_datasets
/nix/store/ps64ljvkg2qh00vk6iina3hb293xlvlk-python3-3.12.9-env/lib/python3.12/site-packages/libsvmdata/libsvm.py:7: in <module>
    LIBSVM_DATA_HOME = _get_data_home("libsvm")
/nix/store/ps64ljvkg2qh00vk6iina3hb293xlvlk-python3-3.12.9-env/lib/python3.12/site-packages/libsvmdata/abstraction.py:21: in _get_data_home
    return data_home / subdir_name
E   TypeError: unsupported operand type(s) for /: 'str' and 'str'
```

--- a/libsvmdata/abstraction.py
+++ b/libsvmdata/abstraction.py
@@ -14,12 +14,14 @@ def _get_data_home(subdir_name=""):
     seek for the variable $XDG_DATA_HOME. Finally, the fallback is $HOME/data.
     """
     data_home = os.environ.get("LIBSVMDATA_HOME", None)
+
     if data_home is None:
         data_home = os.environ.get("XDG_DATA_HOME", None)
     if data_home is None:
         data_home = Path.home() / "data"
     else:
         data_home = Path(data_home)
+
     return data_home / subdir_name if subdir_name else data_home
 
 

--- a/libsvmdata/abstraction.py
+++ b/libsvmdata/abstraction.py
@@ -22,7 +22,7 @@ def _get_data_home(subdir_name=""):
     else:
         data_home = Path(data_home)
 
-    return data_home / subdir_name if subdir_name else data_home
+    return data_home / subdir_name
 
 
 class AbstractDataset(ABC):

--- a/libsvmdata/abstraction.py
+++ b/libsvmdata/abstraction.py
@@ -18,7 +18,9 @@ def _get_data_home(subdir_name=""):
         data_home = os.environ.get("XDG_DATA_HOME", None)
     if data_home is None:
         data_home = Path.home() / "data"
-    return data_home / subdir_name
+    else:
+        data_home = Path(data_home)
+    return data_home / subdir_name if subdir_name else data_home
 
 
 class AbstractDataset(ABC):

